testing/selenium_server.md

```diff
@@ -0,0 +1,94 @@
+## Selenium Server Setup and Testing
+
+Using Selenium Server and a guest virtual machine requires a lot of setup and configuration. The following sections describe the configuration and test execution process when using Selenium Server.
+
+### VirtualBox and Virtual Machine Setup
+
+The following steps describe the process for configuring an Internet Explorer 9 and Windows 7 VM provided by Microsoft.
+
+1. [Download a virtual machine](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/) from Microsoft by selecting 'IE9 on Win7' with 'VirtualBox' as the platform.
+
+2. Import the virtual machine with VirtualBox.
+
+3. Configure your VirtualBox preferences host-only adapter.
+    - [Open](/testing/virtualbox_hostonly.png) Host-only Network `vboxnet0` for [configuration](/testing/virtualbox_hostonly_config.png)
+    - Change the IPv4 Address to `10.100.1.1`
+    - Change the IPv4 Network Mask to `255.255.255.0`
+
+4. Change the virtual machine network configuration.
+    - Adapter 1
+        * Attached to: `Host-only Adapter`
+        * Name: `vboxnet0`
+    - Adapter 2
+        * Attached to: `Bridged Adapter`
+        * Name: `en0: Wi-Fi (AirPort)` (or other wireless network)
+        * Adapter Type: `Intel PRO/1000 MT Desktop (82540EM)`
+
+5. Boot the virtual machine.
+
+6. Setup Selenium Server on the virtual machine.
+    - Create the directory `C:\selenium`
+    - Download the latest version of Java
+    - Download the `selenium-server-standalone-2.53.1.jar` from [here](http://selenium-release.storage.googleapis.com/index.html?path=2.53/) and extract it to `C:\selenium`
+    - Download the `IEDriver` from [here](http://www.seleniumhq.org/download/) and extract it to `C:\selenium`
+    - Modify your `%PATH%` to include `C:\selenium`
+
+7. Configure port-forwarding on the virtual machine.
+    - Open PowerShell running as an Administrator
+    - Run this command
+        * `listenaddress` represents the `SELENIUM_SERVER_APP_DOMAIN`, generally `localhost`
+        * `listenport` represents the port you expose using Selenium Server
+        * `connectaddress` represents the host IP address (our host-only adapter IP address)
+        * `connectport` represents the `SELENIUM_SERVER_APP_PORT`
+          ```bash
+          $ netsh interface portproxy add v4tov4 listenport=3001 listenaddress=localhost connectport=3001 connectaddress=10.100.1.1
```

Comment:
Realize that it's for setting up Linux, but might provide some useful information about app setup that I would guess is shared